pkg/terraform/gather/ovirt/ip.go

```diff
@@ -2,13 +2,113 @@
 package ovirt
 
 import (
+	//"context"
+	"fmt"
 	"github.com/openshift/installer/pkg/terraform"
+	"github.com/openshift/installer/pkg/asset/installconfig/ovirt"
+	ovirtsdk4 "github.com/ovirt/go-ovirt"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"github.com/sirupsen/logrus"
+	"net"
+	"time"
+
+	//"time"
 )
 
+func check_port_is_open(host string, port string) bool {
+		timeout := time.Second
+		conn, err := net.DialTimeout("tcp", net.JoinHostPort(host, port), timeout)
+		if err != nil {
+			fmt.Println("Connecting error:", err)
+		}
+		if conn != nil {
+			defer conn.Close()
+			fmt.Println("Opened", net.JoinHostPort(host, port))
+			return true
+		}
+		return false
+}
+
+func findVirtualMachineIP(c *ovirtsdk4.Connection,moRefValue string) (string, error) {
+
+	vmsService := c.SystemService().VmsService()
```

Comment:
I think tou can do it in one shot with:
vmResp , err := c.SystemService().VmsService().VmService(moRefValue).Get().Send()
if err != nil {
		return "",fmt.Errorf("Failed to find VM, by id %v, reason: %v\n",moRefValue, err)
	}
VM := vmResp.MustVm()